plugins/linux-capture/xcomposite-input.c

```diff
@@ -360,27 +360,6 @@ void xcomp_cleanup_pixmap(Display *disp, struct xcompcap *s)
 	}
 }
 
-static enum gs_color_format gs_format_from_tex()
```

Comment:
This function was called by `xcomp_create_pixmap` at 1604400e48. Later, the call was removed by ddb692ab2.